torch/quantization/quantize_fx.py

```diff
@@ -17,6 +17,97 @@ def _check_is_graph_module(model: torch.nn.Module) -> None:
             'Got type:' + str(type(model)) + ' Please make ' +
             'sure to follow the tutorials.')
 
+
+def _check_is_valid_qconfig_dict(qconfig_dict: Any) -> None:
```

Comment:
can we move these functions here (https://github.com/pytorch/pytorch/blob/master/torch/quantization/fx/qconfig_utils.py) instead, to keep LOC of quantize_fx.py manageable?

Reply:
Sure! Should I also move _check_is_graph_module (lines 13-18) to qconfig_utils.py?

Reply:
great!

> Sure! Should I also move _check_is_graph_module (lines 13-18) to qconfig_utils.py?

sure, moving that to a utils file makes sense as well, with a couple of caveats:
* would be great to have that be a separate PR (since it's not technically related to this PR).  Also, this is optional so you are welcome to do this if you'd like but it's not required.
* since that util is not related to qconfig, maybe `torch/quantization/fx/utils.py` would be a good place for it